games/plusminus/champions/cxx/prologin.hh

```diff
@@ -82,15 +82,16 @@ struct hash<std::vector<T>>
 };
 } // namespace std
 
-extern "C" {
+extern "C"
+{
 
-/// Function called at the start of the game
-void init_game();
+    /// Function called at the start of the game
+    void init_game();
```

Comment:
Maybe we want to add an option to not indent extern C. The same way we don’t indent namespaces.

Reply:
Done.